adkf/gp.py

```diff
@@ -15,7 +15,7 @@ def standard_positive_transform(x: jnp.array) -> jnp.array:
 
 def standard_inverse_transform(x: jnp.array) -> jnp.array:
     """Implements inverse softplus transform."""
-    return jnp.log(jnp.exp(x) - 1)
+    return jnp.log(jnp.exp(x - 1e-6) - 1)
```

Comment:
Maybe add a comment saying this is to under the addition of 1e-6 in the forward transform? Better yet, maybe define `EPS = 1e-6` and use it in both places?